ddspipe_core/include/ddspipe_core/types/dds/TopicQoS.hpp

```diff
@@ -146,6 +146,8 @@ TopicQoS
      */
     HistoryDepthType history_depth = HISTORY_DEPTH_DEFAULT;
 
+    static constexpr HistoryDepthType HISTORY_DEPTH_DEFAULT = 5000;
```

Comment:
We follow the convention of having static methods and attributes grouped apart after non-static ones.